app/src/main/res/values/strings.xml

```diff
@@ -1080,4 +1080,5 @@ If you are overwhelmed by the number of quests, you can always fine-tune which q
     <string name="quest_presets_default_name">Default</string>
     <string name="quest_presets_selected">Selected</string>
     <string name="quest_presets_delete_message">"Delete preset \"%s\"? The quest selection and order for it will be irrevocably lost."</string>
+    <string name="quest_picnicTableCover_title">Is this picnic table protected from the rain?</string>
```

Comment:
> Is the question wording the same as for the bus stop?

No. There it's "Is there a shelter at this bus stop?", but it adds `shelter=yes`, not `covered=yes`. For bike/motorcycle parking it's "Is this bicycle/motorcycle parking covered (protected from rain)?", so let's update the picnic table wording accordingly:


```suggestion
    <string name="quest_picnicTableCover_title">Is this picnic table covered (protected from rain)?</string>
```